swiftly/client/standardclient.py

```diff
@@ -558,10 +560,7 @@ def request(self, method, path, contents, headers, decode_json=False,
                     value = resp.read()
                     resp.close()
             except Exception as err:
-                status = 0
-                reason = '%s %s' % (type(err), str(err))
-                hdrs = {}
-                value = None
```

Comment:
This would change behavior, which worries me. It would mean that all exceptions are no longer converted into HTTPException instances. It'd mean the verbose call, conn.close(), and reset_func would no longer be called. And it means no retries would occur. If the original stack trace is important, perhaps it'd be better to capture that and pass it within the resulting HTTPException?